crates/control/src/behavior/module.rs

```diff
@@ -23,28 +37,174 @@ pub struct CycleContext {
     pub kick_targets: AdditionalOutput<Vec<Point2<f32>>, "kick_targets">,
     pub path_obstacles: AdditionalOutput<Vec<PathObstacle>, "path_obstacles">,
 
-    pub camera_matrices: RequiredInput<Option<CameraMatrices>, "camera_matrices?">,
-    pub projected_limbs: RequiredInput<Option<ProjectedLimbs>, "projected_limbs?">,
+    pub world_state: Input<WorldState, "world_state">,
 
-    pub behavior: Parameter<BehaviorConfiguration, "control.behavior">,
+    pub config: Parameter<BehaviorConfiguration, "control.behavior">,
```

Comment:
Please don't abbreviate 

```suggestion
    pub configuration: Parameter<BehaviorConfiguration, "control.behavior">,
```

Reply:
:wink: done